packages/x-data-grid/src/models/api/gridCoreApi.ts

```diff
@@ -77,10 +77,6 @@ export interface GridCorePrivateApi<
    * The React ref of the grid column container virtualized div element.
    */
   columnHeadersContainerElementRef?: React.RefObject<HTMLDivElement>;
```

Comment:
We could refactor this one to `columnHeadersContainerRef`, long names aren't great for readability.